openreview/venue/venue.py

```diff
@@ -450,7 +450,10 @@ def get_rejected_submission_venue_id(self, submission_invitation_name=None):
         if self.submission_stage:
             return f'{self.venue_id}/Rejected_{self.submission_stage.name}'
         return f'{self.venue_id}/Rejected_Submission'
-
+    
+    def get_active_venue_ids(self, submission_invitation_name=None):
+        return [self.venue_id, self.get_submission_venue_id(submission_invitation_name), self.get_rejected_submission_venue_id(submission_invitation_name)]
```

Comment:
I'm missing why these venue IDs are considered active, I thought the submission venue ID is considered active? But we're including the accepted and rejected venue IDs for a paper (`self.venue_id` and `get_rejected_submission_venue_id`).